ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -10,12 +15,6 @@ use ark_std::{
     vec::Vec,
 };
 
```

Comment:
Not sure why there was a blank line here, I thought it made more sense to remove it, then ran cargo fmt.